setup.py

```diff
@@ -9,8 +9,8 @@
 
 PY_VER = sys.version_info
 
-if PY_VER < (3, 4):
-    raise RuntimeError("aiopg doesn't support Python earlier than 3.4")
+if PY_VER < (3, 5, 2):
```

Comment:
Here there is an interesting point.
I suggest pinning to 3.5.3+

Reply:
Hm.. why?  As I noted on the bug, Ubuntu 16.04 (which is widely used), has Python 3.5.2, there is no newer Python package available for 16.04, not even in xenial-updates or xenial-backports.

So, unless there is a strong reason for requiring 3.5.3, I would prefer to require only 3.5.2, for the time being.

Reply:
@asvetlov  what is interesting point?

Reply:
@gjcarneiro there's a deadsnakes PPA for that one.

Reply:
I am aware of the deadsnakes PPA, it's a not a proper solution.  These packages don't have the same guarantees of being maintained and receiving security updates.

I am not aware of any reason why 3.5.3 is needed.  AFAIK, 3.5.2 is enough.  So I'm going to resolve this for now.

Reply:
@gjcarneiro https://stackoverflow.com/a/52048726/595220

Reply:
Ah, I see, thanks for the link.  Still:
1. Most code doesn't call `asyncio.new_event_loop()`.  If you only call `asyncio.get_event_loop()`, like most asyncio programs do, you have no problem;
2. If this was such a big problem, aiopg would already be currently having it, since right now it requires only Python 3.4;
3. If requiring 3.5.3 is wanted, I ask that someone else make a new PR, as that is a change that is orthogonal to this PR.

Reply:
aiohttp requires it, there's no point in allowing lower version in companion libs

Reply:
But aiopg is not merely a "companion of aiohttp", plenty of use cases for requiring aiopg but not aiohttp.

Reply:
That said, if requiring 3.5.3 is what it takes to unblock this PR, so be it.  But personally I think it's a bad idea.  Too soon.